server/data/data-api/src/test/java/org/apache/james/rrt/lib/MappingTest.java

```diff
@@ -90,6 +96,12 @@ void detectTypeShouldReturnForwardWhenForwardPrefix() {
             .isEqualTo(Type.Forward);
     }
 
+    @Test
+    void detectTypeShouldReturnAliasWhenAliasPrefix() {
+        assertThat(Mapping.detectType(Type.Alias.asPrefix() + "mapping"))
+                .isEqualTo(Type.Alias);
```

Comment:
be careful with indent, after a line break, should be one level of indent. Here is two :)
and some other of tests you added

Reply:
Conf issue with the IDE... Fixed now, thanks for the notice!